### DISCUSSION TITLE
Question: Are conventional commits not used for this on purpose?

### DISCUSSION THREAD

**Discussion by cpolzer** [olivierlacan/keep-a-changelog#611] [Q&A]
Hello,

I am looking into changelog generating and curating solutions currently, for me the question arises if conventional commits that are used from tools like commitizen are not used here on purpose? 

Especially since I was directed to here from a sphinx-needs plugin.

Thanks in advance, I hope my question is delivered in the correct medium...

Regards, Christian

**Comment by willejs** [comment]
What are your thoughts @olivierlacan ?

**Comment by tessus** [comment]
Yes, I think conventional commits tried to tackle the changelog topic. And there is nothing wrong with parsing them and group commits into the conventional commit types: feature, fixes, documentation, ci, tests, ...
The author of keepachangelog added to the FAQ that this project aims to be a better changelog convention.
But this doesn't mean that you can't do it differently. IMO as long as your changelog is consistent and people like it (as in they can read it easily and understand it), it is already a better changelog as what a lot of people/projects do.
